api/server.js

```diff
@@ -29,8 +32,27 @@ const createServer = async () => {
     }
   });
 
+
+
   server.ext('onPreResponse', preResponseUtils.handleDomainAndHttpErrors);
 
+  server.events.on('response', function (request) {
+
+    const trackId = uuid.v1();
```

Comment:
Pourquoi générer un nouvel ID au lieu d'utiliser celui fourni par Scalingo et qui permet de corréler avec les logs routeur ?

```suggestion
    const trackId = request.headers['x-request-id'];
```

Reply:
Bonne idée, je vais l'utiliser.  Il me semble que c'est sans le 'x-'.

Reply:
Done